code/modules/clothing/under/syndicate.dm

```diff
@@ -67,6 +70,7 @@
 	icon_state = "camogreen"
 	inhand_icon_state = "g_suit"
 	can_adjust = FALSE
+	worn_icon = 'icons/mob/clothing/under/syndicate.dmi' //SKYRAT EDIT - ADDITION
```

Comment:
couldnt you just do what #6450 done but with worn icon?
and point the icons/mob/clothing/under/syndicate to uniform dmi in our modular stuff and make this fully modular?

Reply:
I can try to, yeah.